src/PyCall.jl

```diff
@@ -286,6 +286,20 @@ end
 
 getindex(o::PyObject, s::Symbol) = convert(PyAny, getindex(o, string(s)))
 
+function getproperty(o::PyObject, s::Symbol)
+    if s == :o # we still want to be able to access o.o
```

Comment:
`===`

I think we want to deprecate `o.o`, however, in favor of a `PyPtr(o)` method (and an `unsafe_convert(PyPtr, o)` method for `ccall`).   Otherwise you won't be able to access fields named `o` in Python objects.